src/index.html

```diff
@@ -5,18 +5,27 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./style.css">
+  <script src="https://kit.fontawesome.com/e83d856101.js" crossorigin="anonymous"></script>
   <title>To-Do-List App</title>
 </head>
 <body>
 <div class="tasks-wrap">
   <h1>Today's To Do</h1>
-  <div class="card">
-    <input class="add-new"value="Add to your list...">
-    <div class="container" id="tasks">
-      <!-- fetch tasks li -->
+  <div class="container">
```

Comment:
### Optional
- [ ] Although the indexes do not follow the current list order, after adding, deleting and editing items. Their indexes are unique because of adding the name of the tasks and the index as seen in the screenshot below
![Screenshot from 2022-11-04 07-48-55](https://user-images.githubusercontent.com/46388113/199921351-b026f589-d84e-4e6b-ac1f-3709b25971d3.png)